src/containers/CreateVacancy/Forms/BasicInfo/BasicInfo.js

```diff
@@ -457,7 +457,15 @@ const basicInformation = (props) => {
 			<Form.Item label='Application Documents' name='applicationDocuments'>
 				<RequiredDocsList name='applicationDocuments' readOnly={readOnly} />
 			</Form.Item>
-
+			<Tooltip title=''>
```

Comment:
You'll want to add a title here. Something along the lines of "Check this box to enable reference collection through this system. Leave unchecked to manually collect references."